webui/react/src/pages/Admin/UserManagement.tsx

```diff
@@ -343,6 +397,16 @@ const UserManagement: React.FC = () => {
         loading={Loadable.isNotLoaded(loadableUsers)}
         rowClassName={defaultRowClassName({ clickable: false })}
         rowKey="id"
+        rowSelection={{
+          columnWidth: '20px',
+          fixed: true,
+          getCheckboxProps: (record) => ({
+            disabled: record.id === currentUser?.id, // disable the current user not to select onself
```

Comment:
not allow user to select oneself for batch actions to avoid accidents such as deactivate oneself.